neonvm/tools/vm-builder/main.go

```diff
@@ -406,33 +532,37 @@ func main() {
 		log.Fatalln(err)
 	}
 
-	// add 'vector.yaml' file to docker build context
+	// add 'cgconfig.conf' file to docker build context
+	b.Reset()
+	_, err = b.WriteString(configCgroup)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	if err = AddToTar(tw, "cgconfig.conf", b); err != nil {
+		log.Fatalln(err)
+	}
+
+	// add 'pgbouncer.ini' file to docker build context
 	b.Reset()
-	_, err = b.WriteString(configVector)
+	_, err = b.WriteString(configPgbouncer)
 	if err != nil {
 		log.Fatalln(err)
 	}
-	if err = AddToTar(tw, "vector.yaml", b); err != nil {
+	if err = AddToTar(tw, "pgbouncer.ini", b); err != nil {
 		log.Fatalln(err)
 	}
 
 	dockerFileTarReader := bytes.NewReader(buf.Bytes())
 
 	buildArgs := make(map[string]*string)
 	buildArgs["DISK_SIZE"] = size
-	var inittabArg string
-	if *useInittab {
-		inittabArg = "yes"
-	}
-	buildArgs["USE_INITTAB"] = &inittabArg
-
 	opt := types.ImageBuildOptions{
 		Tags: []string{
 			dstIm,
 		},
 		BuildArgs:      buildArgs,
 		SuppressOutput: true,
-		NoCache:        false,
+		NoCache:        true,
```

Comment:
Is this necessary? I'd be concerned that this causes a lot of unnecessary re-builds.